README.md

```diff
@@ -1,3 +1,15 @@
+# ask-ai
+The system can answer 2 types of questions:
+
+1. about entity
```

Comment:
well, subdivision request is about some entity too.

i propose to rephrase it into smth like "entity description request; entity subdivision request"

no sure about that, just IMHO